src/spectrecoff/Tree.fs

```diff
@@ -0,0 +1,57 @@
+[<AutoOpen>]
+module SpectreCoff.Tree
+
+open Spectre.Console
+open SpectreCoff.Layout
+open SpectreCoff.Output
+
+type GuideStyle =
+    | Ascii
+    | SingleLine
+    | DoubleLine
+    | BoldLine
+
+type TreeLayout =
+    {  Sizing: SizingBehaviour
+       Guides: GuideStyle }
+
+let defaultTreeLayout: TreeLayout =
+    {  Sizing = Collapse
+       Guides = SingleLine }
+
+let toOutputPayload tree =
+    tree 
+    :> Rendering.IRenderable 
+    |> Renderable
+
+let private applyLayout layout (root: Tree) =
+    match layout.Sizing with 
+    | Expand -> root.Expanded <- true
+    | Collapse -> root.Expanded <- false
+
+    match layout.Guides with
+    | Ascii -> root.Guide <- TreeGuide.Ascii
+    | SingleLine -> root.Guide <- TreeGuide.Line
+    | DoubleLine -> root.Guide <- TreeGuide.DoubleLine
+    | BoldLine -> root.Guide <- TreeGuide.BoldLine
+    root
+
+let attach (nodes: TreeNode list) (node: TreeNode) =
+    nodes 
+    |> List.iter (fun n -> node.AddNode n |> ignore)
+    node
+
+let createNode (content: OutputPayload) (nodes: TreeNode list) = 
+    let node = TreeNode (content |> payloadToRenderable ) 
```

Comment:
`let node = TreeNode(content |> payloadToRenderable)`

Reply:
Refactored the code, no more braces ;)